### ISSUE TITLE
Post training on custom dataset question on the results.

### ISSUE THREAD

**Issue by josyulakp** [Robbyant/lingbot-va#20]
Hello, 

I have followed the instructions and post trained the model from lingbotva-base to generate my custom dataset weights. 

However I am observing that only the wrist camera input seems to be working well, other streams are frozen, may I know what could be causing this bug?  

https://github.com/user-attachments/assets/0d8cb27e-9918-4f72-a81c-4c6793e22f43

Thank you again.

**Comment by zqh0253** [comment]
Thanks for your interest in our work. Can you share more details on your post-training setup (dataset size, learning rate, training loss curve)?

**Comment by josyulakp** [comment]
Thank you for response. I have shared the details of the run including loss and config metrics [here](https://wandb.ai/krishna-j-perceptyne-robots/wan_va?nw=nwuserkrishnaj). 

Overview: 
Its a multitask data set with 3 subtasks in lerobot-v2 format. 

Dataset episodes: 180 (60x3) 
action config: 7 joints + 1 gripper 

please let me know if any more details are required.

**Comment by lilin-hitcrt** [comment]
> Thank you for response. I have shared the details of the run including loss and config metrics [here](https://wandb.ai/krishna-j-perceptyne-robots/wan_va?nw=nwuserkrishnaj).
> 
> Overview: Its a multitask data set with 3 subtasks in lerobot-v2 format.
> 
> Dataset episodes: 180 (60x3) action config: 7 joints + 1 gripper
> 
> please let me know if any more details are required.

---

Thanks for sharing the loss curve. The initial loss looks unusually high. 

For reference, I've attached a loss curve from my post-training - you can see the initial loss in my case is much lower.

Regarding the action dimensions: the 30-dim output follows this order:
- Left arm EEF (7-dim)
- Right arm EEF (7-dim)  
- Left arm joint (7-dim)
- Right arm joint (7-dim)
- Left gripper (1-dim)
- Right gripper (1-dim)

Missing dimensions should be padded with 0. However, even if the action order doesn't match the pretrained model exactly, it shouldn't cause such a high initial loss. Could you check if the pretrained checkpoint is being loaded successfully?

<img width="1113" height="592" alt="Image" src="https://github.com/user-attachments/assets/11a811c3-a683-4d8b-bd3a-0332523154d3" />

**Comment by lilin-hitcrt** [comment]
I found the issue - you changed the action dimension to 8 in your config, which completely alters the model architecture.

You should use 30-dim actions with unused dimensions padded with 0 (same as the provided RoboWin example - please refer to the example carefully):

Left arm EEF (7-dim)
Right arm EEF (7-dim)
Left arm joint (7-dim)
Right arm joint (7-dim)
Left gripper (1-dim)
Right gripper (1-dim)
Also, if you have limited GPUs, set gradient_accumulation_steps to achieve a larger effective global batch size (e.g., 64).

**Comment by josyulakp** [comment]
Thank you very much. I'll correct the problems and post the results.

**Comment by zkailinzhang** [comment]
> Thank you for response. I have shared the details of the run including loss and config metrics [here](https://wandb.ai/krishna-j-perceptyne-robots/wan_va?nw=nwuserkrishnaj).
> 
> Overview: Its a multitask data set with 3 subtasks in lerobot-v2 format.
> 
> Dataset episodes: 180 (60x3) action config: 7 joints + 1 gripper
> 
> please let me know if any more details are required.

wandb link 404：
Looks like you stumbled on an empty page.
If this is the right URL, you might need to ask the page owner to change the privacy permissions or invite you to the team.

[Go to home page](https://wandb.ai/)
